android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationManagerModule.java

```diff
@@ -106,7 +136,14 @@ public void cancelAll()
 	@Kroll.method
 	public void notify(int id, NotificationProxy notificationProxy)
 	{
-		getManager().notify(id, notificationProxy.buildNotification());
+		Notification notification = notificationProxy.buildNotification();
+
+		// targeting Android O or above? create default channel
+		if (notification.getChannelId() == DEFAULT_CHANNEL_ID) {
```

Comment:
The `notification.getChannelId()` method call is API Level 26 only. We need to do an API Level check or else it'll crash on older Android OS versions.